VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionTextUtils.java

```diff
@@ -191,4 +194,27 @@ public static void setEndorsedState(@NonNull TextView target,
         }
 
     }
+
+    /**
+     * Renders hyperlinks in the given html to clickable items when applied to the given TextView.
```

Comment:
You should also mention that this method sets the HTML spans, and sets the text on the text view as well. The method name should also reflect this as much as possible. Also 'html' should be all caps (applies in the `textView` param documentation as well).